pdns/ws-recursor.cc

```diff
@@ -382,9 +382,9 @@ static void apiServerCacheFlush(HttpRequest* req, HttpResponse* resp) {
   DNSName canon = apiNameToDNSName(req->getvars["domain"]);
   bool subtree = (req->getvars.count("subtree") > 0 && req->getvars["subtree"].compare("true") == 0);
 
-  int count = broadcastAccFunction<uint64_t>([=]{return pleaseWipeCache(canon, subtree, 0xffff);});
+  int count = g_recCache->doWipeCache(canon, true, 0xffff);
```

Comment:
```suggestion
  int count = g_recCache->doWipeCache(canon, subtree, 0xffff);
```
Is the change to always do a subtree wipe intentional?

Reply:
Nope, thaniks for spotting